Remotion/Web/UnitTests/Core/UI/Controls/WebTabStripImplementation/Rendering/WebTabStripRendererTest.cs

```diff
@@ -250,7 +250,7 @@ public void TestDiagnosticMetadataRendering ()
       var wrapper = item.GetAssertedChildElement("span", 0);
       var tab = wrapper.GetAssertedChildElement("span", 1);
       tab.AssertAttributeValueEquals(DiagnosticMetadataAttributes.ItemID, _tab0.Object.ItemID);
-      tab.AssertAttributeValueEquals(DiagnosticMetadataAttributes.Content, _noneHotkeyFormatter.GetFormattedText(_tab0.Object.Text));
+      tab.AssertAttributeValueEquals(DiagnosticMetadataAttributes.Content, HtmlUtility.ExtractPlainText(_noneHotkeyFormatter.GetFormattedText(_tab0.Object.Text)));
```

Comment:
```suggestion
      tab.AssertAttributeValueEquals(DiagnosticMetadataAttributes.Content, _noneHotkeyFormatter.GetFormattedText(_tab0.Object.Text).ToPlainTextString());
```
I think this is the correct conversion here. Didn't test it, though.

Reply:
That won't work as the hotkey formatter always returns an HTML `WebString`

Reply:
Oops. Yes.